experiment/autobumper/bumper/bumper.go

```diff
@@ -94,25 +82,32 @@ func (af *fileArrayFlag) Set(value string) error {
 
 // Options is the options for autobumper operations.
 type Options struct {
-	GitHubOrg     string
-	GitHubRepo    string
-	GitHubLogin   string
-	GitHubToken   string
-	GitName       string
-	GitEmail      string
-	RemoteBranch  string
-	OncallAddress string
-
-	BumpProwImages   bool
-	BumpBoskosImages bool
-	BumpTestImages   bool
-	TargetVersion    string
-
-	IncludedConfigPaths fileArrayFlag
-	ExcludedConfigPaths fileArrayFlag
-	ExtraFiles          fileArrayFlag
-
-	SkipPullRequest bool
+	GitHubOrg           string   `yaml:"gitHubOrg"`
```

Comment:
Technically in Go these fields should all have comments since they are exposed. The reason why they were not initially, as I guessed, is that they were documented as flag parsing logic, such as:
```
flag.StringVar(&o.GitHubToken, "github-token", "", "The path to the GitHub token file.")
```

I think you also need to copy/paste the comments here as you have deleted all flag parsing